slither/core/variables/variable.py

```diff
@@ -22,6 +22,7 @@ def __init__(self):
         self._is_constant = False
         self._is_immutable: bool = False
         self._is_reentrant: bool = True
+        self._storage_location: Literal["memory", "storage", "calldata", "default"] = "default"
```

Comment:
Actually, `LocalVariable` class already has `location` property that holds that information. Please use that.